rd_ui/app/scripts/directives/plotly.js

```diff
@@ -121,6 +121,8 @@
   angular.module('plotly', [])
     .constant('ColorPalette', ColorPalette)
     .directive('plotlyChart', function () {
+      var baseHeight = 250;
```

Comment:
[image: Inline image 2]
I'm having the same difficulty.

On Wed, Apr 27, 2016 at 3:45 PM, Chris Kozak <notifications@github.com>
wrote:

> In rd_ui/app/scripts/directives/plotly.js
> <https://github.com/getredash/redash/pull/1007#discussion_r61323291>:
>
> > @@ -121,6 +121,8 @@
> >    angular.module('plotly', [])
> >      .constant('ColorPalette', ColorPalette)
> >      .directive('plotlyChart', function () {
> > +      var baseHeight = 250;
>
> Hello Arik,
> Kumar has gone on vacation and I'm going to pick up where he left off.
> Which alternative approach would you prefer we take? Delegating to Plotly
> sounds more elegant. What do you think?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/getredash/redash/pull/1007/files/303e158eb1484e9694d90ea27face9465a56d3cc#r61323291>
>